src/ui/component/recommendedContent/view.jsx

```diff
@@ -51,14 +51,10 @@ export default class RecommendedContent extends React.PureComponent<Props> {
     const { recommendedContent, isSearching } = this.props;
 
     return (
-      <section className="media-group--list-recommended">
-        <span>Related</span>
-        {recommendedContent &&
-          recommendedContent.map(recommendedUri => (
-            <FileTile hideNoResult size="small" key={recommendedUri} uri={recommendedUri} />
-          ))}
+      <section className="card">
+        <FileList uris={recommendedContent} header={<span>Related</span>} />
         {recommendedContent && !recommendedContent.length && !isSearching && (
-          <div className="media__subtitle">No related content found</div>
+          <div className="help">No related content found</div>
```

Comment:
IMO this isn't help text and we should reserve `help` for content that is specifically aiding/teaching usage (please bring up on standup if you disagree and/or think we haven't been doing this). I think it is better to add a separate style called "empty" or similar for this type of styling. It's fine if they're even styled identically for now, but using distinct names will ease later style revisions.